File: Java/fibonacci.java
import java.util.Scanner;

class fibonacci{
		public static void main(String args[])
		{
			int n1=1,n2=1,n3;
			int range;
			Scanner sc = new Scanner(System.in);
			System.out.println("Enter a range");
			range=sc.nextInt();
			System.out.print(n1);
			System.out.print(n2);
			while(range>2)
			{
				n3=n1+n2;
				System.out.print(n3);
				n1=n2;
				n2=n3;
				range--;	
			}
		
		}
}

File: Python/fibonacci.py
def  fibonacci(n):

	if n == 1  or n == 0:

		return n;

	else:

		return fibonacci(n-2) + fibonacci(n - 1)


num = int(input("Enter a positive integer : "))

if num < 0:
	print("Number is not valid")

i = 0

print("fibonacci secession: ")

for i in range(0, num):
	print(fibonacci(i))

# Crafted by Sagnik Sahoo © 2022
